web-ui/src/main/resources/catalog/components/edit/bounding/BoundingDirective.js

```diff
@@ -164,17 +165,16 @@
             // parse initial input coordinates to display shape
             ctrl.initValue = function() {
               if (ctrl.polygonXml) {
+                var srsName = ctrl.polygonXml.match(new RegExp("srsName=\"(EPSG:[0-9]*)\""));
+                ctrl.dataProjection = srsName && srsName.length === 2 ?
+                  srsName[1] : 'EPSG:4326';
+
                 // parse first feature from source XML & set geometry name
-                var correctedXml = ctrl.polygonXml
```

Comment:
where do you handle the replace ?

Reply:
https://github.com/geonetwork/core-geonetwork/commit/c5d7ba557fc220a0b0f893e1d4335aaf3122e623